project/AppDependencies.scala

```diff
@@ -7,10 +7,11 @@ object AppDependencies {
   val hmrcMongo = "uk.gov.hmrc.mongo"
   val akka = "com.typesafe.akka"
   val mongoVersion = "0.73.0"
+  val bootstrapVersion = "7.8.0"
```

Comment:
we should probably try increasing this to at least 7.11.0 to get clear of the http-verbs header bug